xl_auth/user/models.py

```diff
@@ -179,6 +184,8 @@ class User(UserMixin, SurrogatePK, Model):
     created_by_id = reference_col('users', nullable=False)
     created_by = relationship('User', remote_side=id, foreign_keys=created_by_id)
 
+    MAX_ALLOWED_ACTIVE_PASSWORD_RESETS = 2
```

Comment:
I think *User. MAX_ALLOWED_ACTIVE_PASSWORD_RESETS* is [a bit at odds with the rest of the code base](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds), for a number of reasons.

First is we don't use constants anywhere else in the models; e.g. the number of seconds that a grant/access token is valid, what amount of time constitutes "recent", how long before a password reset expires, and so forth.

Second is that it (in my eyes) kinda gives the impression that it's something that could (should?) be changed at will. But in the test case layout is such that we in end2end test for *assert password_resets <= User.MAX_PASSWORD_...* and in model-unit we test for what is specified by #155 (exactly two).

Third thing noticed is that if this should be implemented as a "possibly maybe future configurable", then it would be more "at home" alongside *BCRYPT_LOG_ROUNDS*, *CACHE_TYPE* and friends in `xl_auth.settings.py`.

So, bottom line, proposing to ...
1. replace constant `User.MAX_ALLOWED_ACTIVE_PASSWORD_RESETS` with `2` in the 1 place that the Flask app accesses it + tag the squash-commit with `#155` so it's easy to see where it comes from on git-blame, OR
2. make `MAX_ALLOWED_ACTIVE_PASSWORD_RESETS` into an app config setting (not necessarily overridable at this point)

That's my two cents! But aesthetics apart; LGTM.